### ISSUE TITLE
FPP v8: Pi4 Unable to Create Output BBB48String

### ISSUE THREAD

**Issue by MrPaulAR** [FalconChristmas/fpp#1846]
**NOTE:** IF YOU DO NOT ATTACH YOUR LOG FILES, THEN THERE IS A HIGHER PROBABILITY THE ISSUE WON'T BE RESOLVED.  (Steps at end) 
# Please answer the following:
**FPP Version and Hardware**: 
Pi4 with K8-Pi cape running 8.x-master-275-gbc771770 installed via 8.0 pre alpha 3

**Describe the bug and Steps to reproduce**: 
50 pixels attached to port 1.  No other outputs in use.

I was originally getting this error and another about unable to open fb1.  Since this is a test device I used the system settings menu to reset all config except network, rebooted, and then performed initial setup.

Went to the test page and noticed that only 8 channels were listed (I'm expecting 300 as default config uses 100 pixels on each port).  I then proceeded to go to the channel output page, click the first line, then press F2 to get unique channel numbers.  When I clicked save the Pi rebooted which was unexpected.  

After it came back online each output on the channel outputs page was set to start channel 1 and 0 pixels.  I set the first output to 100 pixels and pressed clone string to fill all fields.  Clicked save then restart.   When viewing the test page it still auto populated "All Local Channels" to be 1 through 8.  Testing does not work from either the test page nor the channel outputs drop-down.

The status page shows this error.
> Could not create output type BBB48String. Check logs for details.

The entire log is attached but looks like there are some missing libraries.

```
2024-05-20 09:37:56.821 (7251) [General] /opt/fpp/src/fppd.cpp:708: Creation of Scheduler, Playlist, and Sequence Complete
2024-05-20 09:37:56.834 (7251) [Plugin] /opt/fpp/src/Plugins.cpp:483: Failed to find shlib libfpp-co-BBB48String.so
2024-05-20 09:37:56.835 (7251) [Plugin] /opt/fpp/src/Plugins.cpp:382: Failed to load plugin: libfpp-co-BBB48String.so     Error: libfpp-co-BBB48String.so: cannot open shared object file: No such file or directory
2024-05-20 09:37:56.835 (7251) [ChannelOut] /opt/fpp/src/channeloutput/ChannelOutputSetup.cpp:328: ERROR Opening BBB48String Channel Output
2024-05-20 09:37:56.835 (7251) [ChannelOut] /opt/fpp/src/channeloutput/ChannelOutputSetup.cpp:348: Determined range needed 0 - 7
2024-05-20 09:37:56.841 (7251) [Command] /opt/fpp/src/command.cpp:70: Initializing Command Module
```
[fppd.log](https://github.com/FalconChristmas/fpp/files/15378369/fppd.log)

**Comment by MrPaulAR** [comment]
For what it's worth I did a clean install and the same errors above are still there.

**Comment by MrPaulAR** [comment]
I've imaged back down to 7.5 and things work as expected.

In both cases I did not restore from a backup and did not push any configs from xLights.  The only change I made to the outputs page was to set the first line to 100 pixels, click clone, click ok, then restart fppd.

**Comment by cybercop23** [comment]
I had the same issue tonight with my show while camping. I was testing and trying to set each port to 500, one prop was 450, and as soon as I cloned all 8 ports, I got greeted with that same message. I also got it later when I tried to lower the brightness on each port. A push from xLights fixed it in both cases.
I think it's related to the way the columns are misaligned and the config may be screwed up.
